packages/docker/src/utils.ts

```diff
@@ -84,6 +86,10 @@ export function sanitize(val: string): string {
   return newNameBuilder.join('')
 }
 
+export function fixArgs(args: string[]): string[] {
+  return shlex.split(args.join(' '))
```

Comment:
Is `powershell` ok with this?

Reply:
If arguments are not POSIX complaint, adding quotes around them would fix the problem